docs/_data/team_members.yml

```diff
@@ -135,3 +135,10 @@
   number_educ: 2
   education1: MSc. in Statistics, ETH Zurich
   education2: B.A. in Economics, University of St. Gallen
+  
+ - name: Kaley Jenney
+  photo: ../images/kaley.jpg
+  info: Technical Associate I
+  email: jenneyk@mit.edu
+  number_educ: 1
+  education1: B.S. in Neuroscience, Emmanuel College
```

Comment:
you may want to update these lines by one more space.

Reply:
Can you confirm what you mean by this? Thank you!

Reply:
I just updated the lines by one more space.